CityOfDetroit/bloom
# fix: add mshda 2024 chart

# Pull Request Template

## Issue Overview

This PR addresses #1655 


## Description

This PR uses the ami import script (with some minor updates) to include the AMI chart in the issue.

## How Can This Be Tested/Reviewed?

Reseed with cd backend/core and yarn db:reseed:detroit. Verify the chart is reflected when adding a unit group on the partners side and spot check some values in the db.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
